addons/website_portal_sale/security/ir.model.access.csv

```diff
@@ -14,3 +14,5 @@ access_product_attribute_portal,product.attribute portal,product.model_product_a
 access_product_attribute_value_portal,product.attribute value portal,product.model_product_attribute_value,base.group_portal,1,0,0,0
 access_product_attribute_price_portal,product.attribute price portal,product.model_product_attribute_price,base.group_portal,1,0,0,0
 access_product_attribute_line_portal,product.attribute line portal,product.model_product_attribute_line,base.group_portal,1,0,0,0
+access_product_product_portal,product.product.portal,product.model_product_product,base.group_portal,1,0,0,0
+access_product_template_portal,product.template.portal,product.model_product_template,base.group_portal,1,0,0,0
```

Comment:
Need to give access for portal users to display product image.
see here: https://github.com/odoo/odoo/blob/master/odoo/addons/base/security/ir.model.access.csv#L49